data/sql/updates/pending_db_world/rev_1598881542523907700.sql

```diff
@@ -0,0 +1,10 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1598881542523907700');
+/*
+ * General: Dalaran Fountain
+ * Update by Knindza | <www.azerothcore.org>
+ * Copyright (C) <www.shadowburn.net> & <www.lichbane.com>
+*/
+
+/* Content 3.3.5 */
+
+REPLACE INTO `gameobject` (`guid`, `id`, `map`, `spawnMask`, `phaseMask`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation0`, `rotation1`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`, `VerifiedBuild`) VALUES (268968, 191446, 571, 1, 1, 5804.72, 640.837, 647.627, -0.680677, 0, 0, -0.333807, 0.942641, 180, 255, 1, 0);
```

Comment:
better to use DELETE/INSERT instead of REPLACE

Reply:
Sorted.